docs/migration/enact/migrating-to-enact-4.md

```diff
@@ -10,13 +10,61 @@ this guide.
 
 ## General Changes
 
+### React and React DOM
+Enact 4.0 updates the `react` and `react-dom` dependencies to 17.x.  Developers should ensure
+their code does not rely on features that are no longer available in these versions.
+React 17 introduced the new JSX Transform. The big change is, with the new JSX Transform,
+you can use JSX without importing React. Enact CLI 4 supports this feature on related commands
+and you will now see below lint warning after packing or linting if you didn't change your code.
+
+```js
+warning  'React' is defined but never used  no-unused-vars
+```
+
+Well, you may see lots of them! But don't be afraid. Fortunately, React provides a module that automatically converts the code on behalf of you. It's "react-codemod". Please see more details on [here](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports.).
```

Comment:
please remove `.` on the link
(https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports.).

Reply:
👍